### PR TITLE
[Filter/Sub] Add alias option for backward compatibility.

### PR DESCRIPTION
**Pull request by again4you**
This patch newly adds the `filter-aliases` section for searching the sub-plugin. If the alias name is provided from ini file, then it could be used as sub-plugin.

* Searching sequence: exact match -> priority option ->  `filter-aliases` section

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### Submit v2
* Add `filter-aliases` section in ini file
* Add default `trix-engine` value for backward compatibility of SRNPU.

#### ini file
```
[filter-aliases]
trix-engine = srnpu
```

### CODE REVIEW

**Review by myungjoo — CHANGES_REQUESTED** [review]
This is not general or extensible approach.
We need something that can be applied to any subplugins without the need for recompiling.

File: meson_options.txt [hunk 1]
```diff
@@ -44,6 +44,7 @@ option('framework-priority-tflite', type: 'string', value: 'tensorflow-lite,nnfw
 option('framework-priority-nb', type: 'string', value: '', description: 'A comma separated prioritized list of neural network frameworks to open a .nb file')
 option('framework-priority-bin', type: 'string', value: '', description: 'A comma separated prioritized list of neural network frameworks to open a .bin file')
 option('skip-tflite-flatbuf-check', type: 'boolean', value: false, description: 'Do not check the availability of flatbuf for tensorflow-lite build. In some systems, flatbuffers\' dependency cannot be found with meson.')
+option('srnpu-alias', type: 'string', value: 'trix-engine', description: 'The alias name list of SRNPU sub-plugin. This option provides backward compatibility.')
```
Comment (jaeyun-jung) [inline meson_options.txt:47]: Q: changing 'srnpu-alias' to 'trix-engine-alias'?

fw name is trix-engine and srnpu is option.
then, 'trix-engine-alias' value: 'srnpu' in meson_option and the group name in ini is [trix-engine]

**Comment by myungjoo** [comment]
Interface Suggestion of ```nnstreamer.ini```: e.g.,
```
[filter-aliases]
tf = tensorflow
tf2 = tensorflow
c2 = caffe2
카페2 = caffe2
까페2 = caffe2
```

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**